paddle/fluid/pybind/protobuf.cc

```diff
@@ -205,11 +205,7 @@ void BindBlockDesc(pybind11::module *m) {
 void BindVarDsec(pybind11::module *m) {
   pybind11::class_<pd::VarDesc> var_desc(*m, "VarDesc", "");
   var_desc
-      .def("name",
-           [](pd::VarDesc &self) {
-             pybind11::bytes name = self.Name();
-             return name;
-           },
+      .def("name", [](pd::VarDesc &self) { return self.Name(); },
```

Comment:
```
var_desc.def("name", &pd::VarDesc::Name);
```

Reply:
Will ```pybind11::return_value_policy::reference``` still take effect if I change to ```&pd::VarDesc::Name```

Reply:
OK, I got it, Thanks, I will change it later